post_edit_docs.py

```diff
@@ -34,9 +34,12 @@ def newPathMarkdown():
     soup = BeautifulSoup(html, 'html.parser')
 
     for e in soup.find_all("code"):
-        if len(e) == 1 and e.text.startswith(http_methods):
+        logging.info("newPathMarkdown: processing: '%s'" % (e.text))
+        text = e.text.lstrip()
+        text = text[0:text.find("\n")]
```

Comment:
wouldn´t `e.text.strip()` suffice here? or at least `text.rstrip('\n')`? otherwise I would imagine a text like `abc\n-important part\n` to miss an important part...?

Reply:
> wouldn´t `e.text.strip()` suffice here? or at least `text.rstrip('\n')`? otherwise I would imagine a text like `abc\n-important part\n` to miss an important part...?

thanks Manuel. after short discussion: [fixed](https://github.com/mendersoftware/autodocs/pull/6/commits/f349818dae4f994fbecfcf965bcd09390edf7863#diff-6f74f0b7c8ebe7ceddc1d6c4c20c5096R38).